.github/workflows/visual-regression.yml

```diff
@@ -18,29 +18,29 @@ jobs:
     if: github.event.ref != 'refs/heads/main'
     steps:
       - uses: actions/checkout@v4
-      # - uses: actions/setup-node@v3
-      #   with:
-      #     node-version: 18
-      #     cache: 'npm'
-      # - name: "Download artifacts"
-      #   # Turns out you can not use artifacts from a previous workflow run.
-      #   # I'm reluctant to check in the resulting screenshots into the repository,
-      #   # which is why we use the following hack, until it's solved on Github side:
-      #   # https://github.com/actions/download-artifact/issues/3
-      #   run: |
-      #     RUN_ID=`gh run --repo $GITHUB_REPOSITORY --branch main list --workflow "Visual Regressions" --json databaseId --jq .[0].databaseId`
-      #     echo "Downloading snapshots created in run ${RUN_ID}"
-      #     gh run --repo $GITHUB_REPOSITORY download ${RUN_ID} -n "visual-regression-snapshots" -D ${{ env.VISUAL_REGRESSION_SNAPSHOT_DIRECTORY }}
-      #   env:
-      #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      # - run: npm install
-      # - run: npm run build
-      # - run: npm run test:visual
-      # - uses: actions/upload-artifact@v3
-      #   if: always()
-      #   with:
-      #     name: visual-regression-snapshots-results
-      #     path: ${{ env.VISUAL_REGRESSION_SNAPSHOT_DIRECTORY }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: "npm"
+      - name: "Download artifacts"
+        # Turns out you can not use artifacts from a previous workflow run.
+        # I'm reluctant to check in the resulting screenshots into the repository,
+        # which is why we use the following hack, until it's solved on Github side:
+        # https://github.com/actions/download-artifact/issues/3
```

Comment:
This issue is resolved on github and now supported in version 4: 
https://github.com/actions/download-artifact?tab=readme-ov-file#download-artifacts-from-other-workflow-runs-or-repositories